sdk_container/src/third_party/coreos-overlay/coreos-base/coreos-sb-keys/coreos-sb-keys-0.0.3.ebuild

```
@@ -0,0 +1,29 @@
+# Copyright (c) 2015 CoreOS Inc.
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```

Make it EAPI 8, please.

```suggestion
EAPI=8
```